pkg/reconciler/roles.go

```diff
@@ -260,9 +262,15 @@ func handleRoleCreate(
 		role.roleSpec.Name,
 	)
 
+	nativeSystemdSupport := false
+	if handler.globalConfig != nil {
+		nativeSystemdSupport = handler.globalConfig.Spec.NativeSystemdSupport
```

Comment:
Should we get a read lock for this? I need to refresh on go semantics for atomicity of pointer variable read/update, but I _think_ it's not guaranteed atomic by the language spec.

Reply:
Yes we have to. Will do